scripts/markdown-link-check.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Local script runner for recursive markdown-link-check
+# Based on: https://github.com/gaurav-nelson/github-action-markdown-link-check/blob/master/entrypoint.sh
+
+if [ -z "$(docker image list -q markdown-link-check)" ]; then
+  echo "This script requires the markdown-link-check Docker image."
+  echo ""
+  echo "More information: https://github.com/tcort/markdown-link-check"
+  echo ""
+  echo "Steps: "
+  echo "git clone git@github.com:tcort/markdown-link-check.git"
+  echo "cd markdown-link-check"
+  echo "docker build -t markdown-link-check ."
```

Comment:
https://github.com/tcort/markdown-link-check#run-using-docker

How about using `ghcr.io/tcort/markdown-link-check:stable`?

Reply:
Nice :) Last time I failed to pull this image 🤔 Anyway, it works now. Fixed 803f268